### PR TITLE
Feature/add error message when they dont want you to succeed

### PR DESCRIPTION
**Pull request by RossCasey**

### CODE REVIEW

File: app/Http/routes.php [hunk 1]
```diff
@@ -12,5 +12,5 @@
 */
 
 $app->get('/', function () use ($app) {
-    return $app->version();
+    return 'We just seen two dolphins';
```
Comment (Joeoh) [inline app/Http/routes.php:15]: 👌🏽

File: composer.lock [hunk 2]
```diff
@@ -4,8 +4,8 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
```
Comment (Joeoh) [inline composer.lock]: Can you remove and gitignore this file please?

**Review by Joeoh — COMMENTED** [review]
All looks good, little changes plz

**Review by Joeoh — APPROVED** [review]

File: composer.lock (hunk 2 above)
Comment (RossCasey) [reply composer.lock]: 💯

**Pull request merged**